src/jwt/signatureVerifier.js

```diff
@@ -71,13 +71,16 @@ const getJwk = (domain, kid) => {
           k => k.use === 'sig' && k.kty === 'RSA' && k.kid && (k.n && k.e),
         )
         .find(k => k.kid === kid);
+      if (!key) {
+        throw new Error('Key not present');
```

Comment:
Doesn't really matter what message we put here, as it will get caught on the "catch" clause below